dbcon/mysql/ha_mcs_pushdown.cpp

```diff
@@ -718,15 +718,17 @@ create_columnstore_select_handler(THD* thd, SELECT_LEX* select_lex)
         return handler;
     }
 
+    // This block gives false positive in
+    // partitionOptimization/vTpch11.
     // Disable SH processing for UNION in a subquery.
     // This way, the server falls back to using the DH for executing
     // the UNION, if enabled.
-    TABLE_LIST *tbl_ptr = select_lex->get_table_list();
+    /*TABLE_LIST *tbl_ptr = select_lex->get_table_list();
```

Comment:
4 Gagan. Could you look why this block causes false positive in the mentioned test?